ui/app/controllers/clients/client.js

```diff
@@ -5,6 +5,26 @@ import Sortable from 'nomad-ui/mixins/sortable';
 import Searchable from 'nomad-ui/mixins/searchable';
 
 export default Controller.extend(Sortable, Searchable, {
+  // ideally this would concat from the clients controller
```

Comment:
Why doesn't it? If it can't, can you add that explanation in the comment please.